packages/core/src/components/header/header.scss

```diff
@@ -89,25 +112,37 @@
 
 .dso-navbar {
   flex-basis: 100%;
-
-  .dso-dropdown-options {
-    border-block-start: 1px solid colors.$grijs-20;
-  }
 }
 
 dso-dropdown-menu {
+  @include dropdown-menu.children();
```

Comment:
FYI: Dit is dus precies waarom we styling die bij het Web Component hoort, in de stylesheet van het Web Component moeten onderbrengen.

En dus met `::slotted()` selectors aan de bak moeten.

Maar dit slechts ter info.